python/lsst/pipe/supertask/config.py

```diff
@@ -47,6 +47,50 @@
 # ------------------------
 
 
+def InputDatasetField(doc, name, units, storageClass, check=None):
+    """Factory function to create `~lsst.pex.config.Config` class instances
+    of `InputDataSetConfig`
+
+    This function servers as syntactic sugar for creating Configurable fields
+    which are `InputDatasetConfigs`. The naming of this function violates the
+    normal convention of a lowercase first letter in the function name, as
+    this function is intended to sit in the same place as
+    `~lsst.pex.config.ConfigField` classes, and consistency in declaration
+    syntax is important.
+
+    The input arguments for this class are a combination of the arguments for
+    `~lsst.pex.config.ConfigField` and `InputDatasetConfig`. The arguments
+    doc and check come from `~lsst.pex.config.ConfigField`, while name,
+    units, and storageClass come from `InputDatasetConfig`.
+
+    Parameters
+    ----------
+    doc : `str`
+        Documentation string for the `InputDatasetConfig`
+    name : `str`
+        Name of the `~lsst.daf.butler.DatasetType` in the returned
+        `InputDatasetConfig`
+    units : iterable
+        List of DataUnits for this `~lsst.daf.butler.DatasetType`
```

Comment:
Is it a list of `DataUnit` or is it a list of `str` keys?

Reply:
It is list of strings which are DataUnit names/keys.

Reply:
Excellent, so the parameter should say:
```
units : `list` of `str`
```
and that would remove confusion.